src/web/app/components/question-types/question-constraint/contribution-question-constraint.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="row" *ngIf="questionDetails.isZeroSum && !isAllFormsNotAnswered">
+    <div class="col-12">
+        <p *ngIf="isInsufficientSharesDistributed" class="text-danger"><span class="fa fa-times"></span>
+            Actual total is {{ totalAnsweredShares }}! Distribute the remaining {{ totalRequiredShares - totalAnsweredShares }} shares.</p>
```

Comment:
- Indentation should be 2 spaces.
- Try to format the code such that the mental workload of having to locate where the distinct elements are is minimzed. In the current version, it's not so easy to see the hierarchy, and the closing `</p>` is very missable.
```html
<p *ngIf="isInsufficientSharesDistributed" class="text-danger">
  <span class="fa fa-times"></span>
  Actual total is {{ totalAnsweredShares }}! Distribute the remaining {{ totalRequiredShares - totalAnsweredShares }} shares.
</p>
```